src/dialects/sqlite/query-generator-typescript.ts

```diff
@@ -15,6 +15,16 @@ export class SqliteQueryGeneratorTypeScript extends MySqlQueryGenerator {
     return `PRAGMA TABLE_INFO(${this.quoteTable(tableName)});`;
   }
 
+  showConstraintsQuery(tableName: TableNameOrModel, _constraintName?: string) {
+    const table = this.extractTableDetails(tableName);
+
+    return joinSQLFragments([
+      'SELECT sql FROM sqlite_master',
+      `WHERE tbl_name = ${this.escape(table.tableName)}`,
+      ';',
```

Comment:
```suggestion
```
I'm removing the trailing semicolon from here, see #15454

Reply:
Done for all dialects